src/pages/bloggers/BloggerPage.tsx

```diff
@@ -0,0 +1,114 @@
+import { Flex, Grid } from '@chakra-ui/react';
+import { useEffect, useRef, useState } from 'react';
+import { useLocation, useParams } from 'react-router';
+
+import { Hero } from '~/components/shared/blogs/Hero';
+import { NotesList } from '~/components/shared/blogs/NotesList';
+import { OtherBlogsList } from '~/components/shared/blogs/OtherBlogsList';
+import { UiButton } from '~/components/ui/UiButton';
+import { UiCardGrid } from '~/components/ui/UiCardGrid';
+import { DATA_TEST_IDS } from '~/constants/test-ids';
+import { useErrors } from '~/hooks/use-errors';
+import { useLazyGetBloggerByIdQuery } from '~/query/blogs-api';
+import { useLazyGetRecipesByUserIdQuery } from '~/query/recipe-api';
+import { useAppDispatch, useAppSelector } from '~/store/hooks';
+import { selectCurrentUserId } from '~/store/selectors';
+import { setBlogger } from '~/store/user-slice';
+import { ErrorResponse, Recipe } from '~/types';
+
+export const BloggerPage = () => {
+    const { bloggerId } = useParams();
```

Comment:
```suggestion
    const { bloggerId = '' } = useParams();
```
после этого не нужно будет постоянно использовать type assertion, вроде как